server/grpc_ac.go

```diff
@@ -80,19 +80,19 @@ func (s *grpcServer) GetActionResult(ctx context.Context,
 	}
 	for _, of := range result.GetOutputFiles() {
 		_, ok := inlinableFiles[of.Path]
-		err = s.maybeInline(ok, &of.Contents, &of.Digest, &inlinedSoFar)
+		err = s.maybeInline(req.GetInstanceName(), ok, &of.Contents, &of.Digest, &inlinedSoFar)
 		if err != nil {
-			s.accessLogger.Printf("%s %s %s", errorPrefix, req.ActionDigest.Hash, err)
+			s.accessLogger.Printf("%s [%s] %s %s", errorPrefix, req.GetInstanceName(), req.ActionDigest.Hash, err)
 			return nil, status.Error(codes.Unknown, err.Error())
 		}
 	}
 
-	s.accessLogger.Printf("GRPC AC GET %s OK", req.ActionDigest.Hash)
+	s.accessLogger.Printf("GRPC AC GET [%s] %s OK", req.GetInstanceName(), req.ActionDigest.Hash)
 
 	return result, nil
 }
 
-func (s *grpcServer) maybeInline(inline bool, slice *[]byte, digest **pb.Digest, inlinedSoFar *int64) error {
+func (s *grpcServer) maybeInline(instanceName string, inline bool, slice *[]byte, digest **pb.Digest, inlinedSoFar *int64) error {
```

Comment:
I guess the instanceName arg is useless, since these items are stored in the CAS and this patch doesn't separate CAS entries by instance.